linux_os/guide/system/accounts/accounts-session/accounts_tmout/rule.yml

```diff
@@ -5,9 +5,9 @@ prodtype: fedora,ol7,ol8,rhel6,rhel7,rhel8,rhv4,sle15,wrlinux1019
 title: 'Set Interactive Session Timeout'
 
 description: |-
-    Setting the <tt>TMOUT</tt> option in <tt>/etc/profile</tt> ensures that
+    Setting the <tt>TMOUT</tt> option in <tt>/etc/profile.d/tmout.sh</tt> ensures that
     all user sessions will terminate based on inactivity. The <tt>TMOUT</tt>
-    setting in <tt>/etc/profile</tt> should read as follows:
+    setting in <tt>/etc/profile.d/tmout.sh</tt> should read as follows:
     <pre>TMOUT=<sub idref="var_accounts_tmout" /></pre>
```

Comment:
Please put here all three lines used in remediation, as I think that the part making the variable readonly is important.

Reply:
@vojtapolasek Making it readonly is a new rule and check, and it needs to go there and not in this rule.

Reply:
My understanding is that this rule ensures that the TMOUT env variable is configured to some predefined value. The way proposed in latest STIG for rhel7 and also in this pull request ensures it very well, the readonly line makes sure that it is not trivial to overwrite the variable. I find this desirable. What is the reason to separate this into multiple rules?

Reply:
Better support in operational environments which as the vendor we are moving towards. And actually, changing this to a one-liner like `declare -xr TMOUT=600` would be way better. DISA was in favor of this alternative.

Reply:
I am okay with changing the line just to declare -xrj TMOUT= something. @cyarbrough76  @thenefield  is that OK for you? I also suggest that both /etc/profile and /etc/profile.d/tmout.sh are checked, while the remediation ensures that the line exists ONLY in /etc/profile.d/tmout.sh.